src/parser/schema.json

```diff
@@ -877,6 +904,7 @@
     },
     "WebsiteComponent": {
       "additionalProperties": false,
+      "description": "The WebsiteComponent interface is used to define the properties of a website component. A WebsiteComponent is used to render an iframe with a website inside of it. This can be used to display an external website or an html file that is located in the public folder.",
```

Comment:
Are these programmatically generated?

Reply:
Ya I was also wondering why something changed here? I thought maybe i installed a updated version of whatever library did this or something?